src/hooks/useViewportColumns.ts

```diff
@@ -199,7 +204,7 @@ export function useViewportColumns<R, SR>({
     const colOverscanEndIdx = Math.min(lastColIdx, colVisibleEndIdx + 1);
 
     return [colOverscanStartIdx, colOverscanEndIdx];
-  }, [columns, columnMetrics, lastFrozenColumnIndex, scrollLeft, totalFrozenColumnWidth, viewportWidth]);
+  }, [columnMetrics, columns, enableVirtualization, lastFrozenColumnIndex, scrollLeft, totalFrozenColumnWidth, viewportWidth]);
```

Comment:
`enableVirtualization` is unlikely to change often, so we can move it last (micro-optimization)
```suggestion
  }, [columnMetrics, columns, lastFrozenColumnIndex, scrollLeft, totalFrozenColumnWidth, viewportWidth, enableVirtualization]);
```